src/utils/featheryClient/integrationClient.ts

```diff
@@ -352,17 +352,19 @@ export default class IntegrationClient {
       use_quik: action.quik_documents
     };
 
-    const fieldVal = fieldValues[action.quik_tags_field_key];
-
-    if (action.quik_tags_field_key) {
-      if (typeof fieldVal === 'string') {
-        payload.quik_tags = (fieldVal as string)
-          .split(',')
-          .map((tag) => tag.trim());
-      } else if (fieldVal instanceof Array) {
-        payload.quik_tags = fieldVal;
-      } else {
-        payload.quik_tags = [JSON.stringify(fieldVal)];
+    if (action.quik_documents) {
```

Comment:
this code looks good but we also need to set the field_values response from the HTTP request into the fieldValues cache